src/india_api/cmd/main.py

```diff
@@ -1,14 +1,25 @@
 """The main entrypoint to the application."""
 
 import uvicorn
+import sentry_sdk
 
 from india_api import internal
 from india_api.internal.config import Config
 from india_api.internal.service import get_db_client, server
-
+__version__ = "0.1.37"
```

Comment:
I think its better to import this, from where I said, otherwise there are two version and its hard to keep them both update to date

Reply:
yep, made that change